.github/workflows/pr_tests.yml

```diff
@@ -35,6 +35,23 @@ jobs:
 
       - name: Run unit tests
         run: ./gradlew --no-daemon jvmTest
+      - name: Run Detekt (core module)
+        run: ./gradlew --no-daemon --console=plain :core:detekt
+
+      - name: Show Detekt summary
+        if: always()
+        run: |
+          SUMMARY_MD=core/build/reports/detekt/detekt.md
+          SUMMARY_TXT=core/build/reports/detekt/detekt.txt
+          if [ -f "$SUMMARY_MD" ]; then
```

Comment:
In which cases the summary is md vs text? Why do we need both?

Reply:
HTML, MD and TXT all three are getting generated for now. I think we can keep the MD one only for workflow logs